SEP-0009.md

```diff
@@ -3,71 +3,59 @@
+the introduction of new features or API changes but how and when release occur
+and how they are described.
 
 # Detailed Description
 
 ## Release Schedule
 
-This SEP proposes that the SunPy core package will be released at least twice a
-year, initially a release in the months of May and November have been chosen to
-place the SunPy release shortly after that of the Astropy core package. The
-decision to make a release and the specific timing of releases are at the
-discretion of the lead developer, and changes to the months of the releases can
-be made without modifying this SEP to follow any pertinent release cycles of
-upstream packages or any other reason.
-
-Future changes to the LTS release cycle should be informed by upstream package
-development. It is important that upstream packages versions are supported for
-the lifetime of a SunPy LTS.
+This SEP prescribes that there will be at least two releases of the sunpy core
+package per year with at least 4 months (preferably 6 months) between these
+releases. It is suggested that these occur in the months of May and
+November based on the current release schedule of the astropy core package. It
+is the responsibility of the lead developer to create a release schedule that
+is consistent with these requirement and to present it to the Board at the
+beginning of every year.  Releases are recommended to be timed to pertinent
+release cycles of upstream packages.
 
 ## Supporting Releases with bug fixes
 
-This SEP proposes that there be two categories of SunPy release, a LTS (Long
-Term Support) release and a normal or non-LTS release. The LTS release will have
-an extended length of bug fixes provided by the SunPy developers. We propose
-that the first release in a year be a non-LTS release, which will be supported
-for 6 months (until the next release), the second release in a year will be an
-LTS release and will be supported for 12 months (until the next LTS release).
+This SEP defines two categories of SunPy release, a Long
+Term Support (LTS) release and a short-support or non-LTS release. Both types of
+releases will be supported by bug fixes and documentation upgrades but for
+differing amounts of time. The first release
+of the year shall be a non-LTS release, which shall be supported
+for 6 months or until the next release. The second release of the year
+will be an LTS release and will be supported for 12 months or until
+the next LTS release. Support periods can be extended beyond the requirements
+here if needed.
 
 The objective of this support cycle is to balance the desire to provide a
 release cycle for people who wish to experience minimal changes to SunPy with
-minimizing the maintenance burden on the SunPy developers. It is feasible that
-we could extend the support cycle of the LTS release to longer than 12 months,
-to give users an overlap period where they can upgrade. However, at this time,
```

Comment:
I *really* want to keep the "However, at this time, supporting more than two releases simultaneously is beyond the resources of the project." I think that's it is important we make it plainly clear that we can't support this kind of thing given our current (lack of) funding structure.

Reply:
But I really don't think that belongs in an SEP...how about we say something like this release schedule is consistent with the current available resources and should be revisited if more resources become available?

Reply:
Done.